package.json

```diff
@@ -14,21 +14,24 @@
   "version": "3.1.1",
   "main": "index.js",
   "dependencies": {
-    "abstract-leveldown": "0.12.3",
+    "abstract-leveldown": "2.6.1",
     "argsarray": "0.0.1",
     "d64": "^1.0.0",
+    "ltgt": "^2.1.3",
     "tiny-queue": "0.2.0"
   },
   "devDependencies": {
-    "levelup": "^0.18.2",
-    "tape": "^2.12.3"
+    "asyncstorage-mock-another": "^1.0.1",
+    "levelup": "^1.3.0",
+    "react-native": "file:./mock-react-native",
```

Comment:
As a devDependency, it is disguising itself as if it would be `react-native`. When developing this package, we want it to be required so that tests pass. When using it in a project, we don't want to install it, we want `react-native` to refer to the real React Native.